src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,68 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+class Graph {
+    fun f() {
+        return
+        println() // unreachable
+    }
+
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    val user: User? = null
+
+    fun findShortestPath(node1: Node, node2: Node): Int {
+        return 0
+        val x = 3
+        println() // unreachable
+    }
+
+    fun getNodeById(id: String): Node {
+
+        // TODO: ?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/51112b832939022334567e2f7c8f1966949562a4/src/main/kotlin/Graph.kt#L25"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>